frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js

```diff
@@ -43,22 +41,26 @@ const ApproverReviewPage = ({
       <Label htmlFor="status">Choose report status</Label>
       <Dropdown id="status" name="status" defaultValue="" inputRef={register({ required: true })}>
         <option name="default" value="" disabled hidden>- Select -</option>
-        {possibleStatus.map((status) => (
-          <option key={status} value={status}>{status}</option>
+        {managerReportStatuses.map((status) => (
+          <option key={status} value={status}>{_.startCase(status)}</option>
         ))}
       </Dropdown>
       <Button type="submit" disabled={!valid}>Submit</Button>
     </Form>
   </>
 );
 
-ApproverReviewPage.propTypes = {
+Review.propTypes = {
   reviewed: PropTypes.bool.isRequired,
-  additionalNotes: PropTypes.string.isRequired,
+  additionalNotes: PropTypes.string,
   register: PropTypes.func.isRequired,
   valid: PropTypes.bool.isRequired,
   handleSubmit: PropTypes.func.isRequired,
   onFormReview: PropTypes.func.isRequired,
 };
 
-export default ApproverReviewPage;
+Review.defaultProps = {
+  additionalNotes: 'No creator notes',
```

Comment:
This does not appear to display if the additionalNotes are an empty string, as opposed to `undefined` or `null`